<launch>
  <arg name="scene" default="/home/rtu/dataset/habitat/hm3d/hm3d/00009-vLpv2VX547B/vLpv2VX547B.basis.glb"/>
  <arg name="dataset" default="/home/rtu/dataset/habitat/hm3d/hm3d_annotated_basis.scene_dataset_config.json"/>
  
  <param name="habitat_scene" type="str" value="$(arg scene)" />
  <param name="habitat_dataset" type="str" value="$(arg dataset)" />
  
  <rosparam command="load" file="$(find habitat_ros)/config/VLP_16.yaml" />

  <node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" args="0 0 0 -1.57 0 -1.57 camera_link camera 50" />
    
  <node pkg="habitat_ros" type="sim_old.py" name="habitat_simulator" />
</launch>